holoviews/core/data/xarray.py

```diff
@@ -12,6 +12,19 @@
 from .grid import GridInterface
 from .interface import Interface, DataError, dask_array_module
 
+try:
+    import cftime
+    cftime_types = (
+        cftime._cftime.DatetimeGregorian,
+        cftime._cftime.Datetime360Day,
+        cftime._cftime.DatetimeJulian,
+        cftime._cftime.DatetimeNoLeap,
+        cftime._cftime.DatetimeProlepticGregorian
```

Comment:
I think you are missing `cftime.DatetimeAllLeap` here.  That said, all of these are subclasses of `cftime.datetime`, so I think you could get away with just using `cftime.datetime` here, rather than enumerating all of the different subclasses (since `cftime_types` is only used for instance checks).

Reply:
That's much easier thanks.